File: Generic Programming in Cpp/problem9.cpp
// Accept N number from user and display number

#include <iostream>
using namespace std;

template <class T>

void Display(T Arr[], int iCnt)
{
    int i = 0;
    for (i = 0; i < iCnt; i++)
    {
        cout << Arr[i] << endl;
    }
}

int main()
{
    int Brr[] = {10, 20, 30, 40, 50};

    Display(Brr, 5);

    float Frr[] = {10.11, 20.22, 30.33, 40.44, 50.55};

    Display(Frr, 5);

    return 0;
}

File: problems-on-String-in-Java/problem9.java
// Accept string from user and display word which has maximum charecter , from sentence of string....



import java.io.*;
import java.util.*;

class program306 {
    public static void main(String arg[]) 
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter string");
        String str = sobj.nextLine();

        String newstr=str.replaceAll("\\s+", " ");                    //  "\\s+" means replace more than one white space between two words  into   " "  one white space....          
        String newstr2=newstr.trim();                                                    // The trim() method removes whitespace from both ends of a string 
       String arr[]=newstr2.split(" ");

       int max=0;
        int iPos=0;                    // Taken to present index for maximum word.....


       for (int i=0;i<arr.length;i++)
       {
         if(arr[i].length()>max)
         {
            max=arr[i].length();

            iPos=i;
         }
       }

       System.out.println("Length of largest word is :"+max);
       System.out.println("largest word is :"+arr[iPos]);
    }
}